.gitignore

```diff
@@ -7,6 +7,9 @@ yarn-error.log*
 pnpm-debug.log*
 lerna-debug.log*
 
+
+package-lock.json
+
```

Comment:
Please remove this from .gitignore, we want to commit package-lock.json now so that we can npm ci the latest packages.

Reply:
removed